Dockerfile

```diff
@@ -42,9 +42,9 @@ RUN dnf update -y
 RUN dnf install -y which iproute iputils procps-ng vim-minimal tmux net-tools htop jq
 USER root
 
-COPY --from=hermes-builder /usr/bin/hermes /usr/local/bin/
+COPY --from=hermes-builder /usr/bin/hermes /usr/local/bin/hermes
```

Comment:
Why did this change?

Reply:
docker image changes. It was necessary to specify the exact path.